src/access.py

```diff
@@ -30,6 +30,10 @@ def get_access_params(mgmnt_sys):
         options = {'server': url_mgmnt_sys['zenhub_url']}
         path_to_token = token_path['api_token_zenhub']
         logging.info('Accessing ZenHub')
+    elif mgmnt_sys in ['git', 'github']:
+        options = {'server': url_mgmnt_sys['github_url']}
+        path_to_token = token_path['api_token_github']
+        logging.info('Accessing GitHub')
```

Comment:
GitHub token is used in github.py to accomplish functions that the ZenHub api doesn't support, like getting issue description text, timestamps, etc. and posting new issues.

Reply:
The reason why I had asked is because we can get information on issues without a token. Just tried ` response = requests.get('https://api.github.com/repos/DataBiosphere/azul/issues/448') ` as a random example and got a 200. What information can we not get without a token? If none, we should not bother, otherwise leave it as is for now.

Reply:
If you use GitHub (i.e., ran `git init` or something) it will write a file `.gitconfig` into your home directory. It might be confusing if we name the file containing the GitHub API token `.github_config`. If we rename it we should rename all three config files, or put all three token into one file and name it `.sync-agile-board_config` or something. I'm also not yet sure who's token will be used when we run this in a AWS Lambda?

Reply:
It's necessary for 'post' and 'put', without the token header these return 404. 'Get' seems to work without it, but I think it would be necessary for any repo that's not publicly visible.

Reply:
Renamed token files to `.sync-agile-board-<jira/github/zenhub>_config` to clarify